_components/alerts.md

```diff
@@ -5,8 +5,42 @@ title: Alerts
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Alerts_UI_v1.png">
+
+  <div class="usa-alert usa-alert-success usa-sans">
```

Comment:
Is there a reason why we can't put the usa-sans rule in usa-alert?

I feel like I see that class everywhere.